### PR TITLE
Feature: SigProfiler

### PR DESCRIPTION
**Pull request by bryce-turner**
Resolves issue #414

### CODE REVIEW

File: required_scripts/sigprofiler_fdfe4e8.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+    if 'DINUC' in matrices:
+        num_tasks = num_tasks + 1
+        sig_list.append(('DBS', '78'))
+
+    if 'ID' in matrices:
+        num_tasks = num_tasks + 1
+        sig_list.append(('ID', '83'))
+
+    cpus_per_task = int(args.threads/num_tasks)
```
Comment (ChristopheLegendre) [inline required_scripts/sigprofiler_fdfe4e8.py:94]: suggestion: what if the value is -1; maybe add a min() function?

**Review by ChristopheLegendre — CHANGES_REQUESTED** [review]
looks good except the threads == -1 that could potentially lead to an error if the thread value is not set by the user.

File: required_scripts/sigprofiler_fdfe4e8.py (hunk 1 above)
Comment (bryce-turner) [reply required_scripts/sigprofiler_fdfe4e8.py]: Fixed in commit 92f3ba3

**Review by ChristopheLegendre — APPROVED** [review]
looks good

**Pull request merged**